.github/workflows/ci.yml

```diff
@@ -50,6 +50,11 @@ on:
         type: string
         required: false
         default: ""
+      frontend-secrets:
+        description: The secrets to use within frontend steps
+        type: string
+        required: false
+        default: ""
```

Comment:
This needs to be added to `cd.yml` as well (https://github.com/grafana/plugin-ci-workflows/blob/main/.github/workflows/cd.yml), and passed when invoking the ci workflow here: https://github.com/grafana/plugin-ci-workflows/blob/840c01bf99d7d9135c327e528abe1673072d90f5/.github/workflows/cd.yml#L313-L341